src/modules/CRUD/services/attachments.js

```diff
@@ -0,0 +1,16 @@
+import Api from '@terralego/core/modules/Api';
+import { sanitizeCustomEndpoint } from './utils';
+
+const attachmentEndpoint =  (endpoint, body, method = 'PUT') =>
```

Comment:
```suggestion
const attachmentEndpoint = (endpoint, body, method = 'PUT') =>
```

Reply:
Also, I don't think `attachmentEndpoint` is the best name for a function that will in fact do a request.